### PR TITLE
Add endpoint to get all visualizations for a user

### PR DESCRIPTION
**Pull request by cameronmarsh**
smartcolumbusos/discovery_ui#26

### CODE REVIEW

File: lib/discovery_api_web/views/visualization_view.ex [hunk 1]
```diff
@@ -14,6 +14,20 @@ defmodule DiscoveryApiWeb.VisualizationView do
     }
   end
 
+  def render("visualizations.json", %{visualizations: visualizations}) do
+    visualizations
+    |> Enum.map(fn visualization ->
+      %{
```
Comment (bbrewer-papa-x) [inline lib/discovery_api_web/views/visualization_view.ex]: Would it make sense to have this mapping match between the single and multi mappings

File: test/unit/discovery_api_web/controllers/visualization_controller_test.exs [hunk 2]
```diff
@@ -195,6 +201,34 @@ defmodule DiscoveryApiWeb.VisualizationControllerTest do
                "chart" => @decoded_chart
              } = body
     end
+
+    test "GET /visualization gets all visualizations for user with valid id and bearer token", %{subject_id: subject_id, token: token} do
```
Comment (bbrewer-papa-x) [inline test/unit/discovery_api_web/controllers/visualization_controller_test.exs]: Valid id? Good catch on the other descriptions though

Comment (cameronmarsh) [reply test/unit/discovery_api_web/controllers/visualization_controller_test.exs]: Ah, woops I'll fix that

File: lib/discovery_api_web/views/visualization_view.ex (hunk 1 above)
Comment (cameronmarsh) [reply lib/discovery_api_web/views/visualization_view.ex]: Yeah that makes sense

**Pull request merged**